src/include/miopen/solver.hpp

```diff
@@ -415,6 +415,42 @@ struct ConvAsm1x1U final : ConvTunableSolver<PerformanceConfigConvAsm1x1U>
                              const PerformanceConfigConvAsm1x1U&) const override;
 };
 
+struct PerformanceConfigConvBiasActivAsm1x1U : PerformanceConfigConvAsm1x1U
+{
+    PerformanceConfigConvBiasActivAsm1x1U(bool spare) : PerformanceConfigConvAsm1x1U(spare) {}
+    PerformanceConfigConvBiasActivAsm1x1U()
+        : PerformanceConfigConvAsm1x1U(-1, -1, -1, -1, -1, -1, -1, -1, false)
+    {
+    }
+    bool IsValid(const ConvolutionContext& config) const;
+    bool operator==(const PerformanceConfigConvBiasActivAsm1x1U& other) const;
+};
+
+// Fused solver
+struct ConvBiasActivAsm1x1U final : ConvTunableSolver<PerformanceConfigConvBiasActivAsm1x1U>
+{
+    const std::string& SolverDbId() const override { return GetSolverDbId<ConvBiasActivAsm1x1U>(); }
+
+    bool IsApplicable(const ConvolutionContext& params,
+                      const ProblemDescription& problem) const override;
+    size_t GetWorkspaceSize(const ConvolutionContext& params,
+                            const ProblemDescription& problem) const override;
+    bool MayNeedWorkspace() const override { return true; }
+
+    PerformanceConfigConvBiasActivAsm1x1U
+    GetDefaultPerformanceConfig(const ConvolutionContext&,
+                                const ProblemDescription&) const override;
+    bool IsValidPerformanceConfig(const ConvolutionContext&,
+                                  const ProblemDescription&,
+                                  const PerformanceConfigConvBiasActivAsm1x1U&) const override;
+    PerformanceConfigConvBiasActivAsm1x1U Search(const ConvolutionContext&,
+                                                 const ProblemDescription&,
+                                                 const AnyInvokeParams& invoke_ctx) const override;
+    ConvSolution GetSolution(const ConvolutionContext& params,
+                             const ProblemDescription& problem,
+                             const PerformanceConfigConvBiasActivAsm1x1U& config) const override;
+};
```

Comment:
This change should not be here

Reply:
Perhaps a merge conflict, maybe merging the base branch into this branch will fix this issue.